src/Microsoft.DotNet.CodeFormatter.Analyzers/Resources.resx

```diff
@@ -126,6 +126,15 @@
   <data name="ExplicitThisFixer_Title" xml:space="preserve">
     <value>Remove 'this' qualifier</value>
   </data>
+  <data name="ExplicitVariableTypeAnalyzer_MessageFormat" xml:space="preserve">
+    <value>Variable '{0}' should be declared with explicit type</value>
+  </data>
+  <data name="ExplicitVariableTypeAnalyzer_Title" xml:space="preserve">
+    <value>Replace the usage of 'var' with an explicit type</value>
+  </data>
+  <data name="ExplicitVariableTypeFixer_Title" xml:space="preserve">
+    <value>Replace the usage of 'var' with explicit type</value>
```

Comment:
"... with *an* explicit type" (for consistency with the fixer title). And I think the message format should also be ".. with *an* explicit type".